docs/tools/cli.rst

```diff
@@ -63,6 +63,8 @@ Manage the payment cards on your account.
 
```

Comment:
Could we also make the `cloud `heading linked?

Reply:
If we do so we have to change all the titles from being code like ``cloud`` to cloud. @lornajane what do you think?

Reply:
The drawback if we don't link from the title is that we'll need to add aSee detailed command information .... for each command.. that could become quite repetitive

Reply:
Let's leave it as it is. Linking titles isn't a common UX pattern (or the links would be internal ones to let you link directly to the section, not to somewhere else) and I think people should read the high level information and then decide if they need more info.